.github/workflows/workflow.yml

```diff
@@ -248,7 +271,50 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0 # Fetch all the tags
+      - name: Get version
+        run: |
+          chcp 65001 #set code page to utf-8
+          echo ("PACKAGE_VERSION=" + (python3 ./setup.py --version) ) >> $env:GITHUB_ENV
```

Comment:
Is this not shell script? If so, I don't understand the syntax.

Reply:
It is Windows PowerShell code, not Linux Bash.
It is the way to set an env variable in GitHub Action : https://stackoverflow.com/a/66737579/14541668